cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/LoadMonitor.java

```diff
@@ -490,6 +527,29 @@ public ClusterModel clusterModel(long from,
           clusterModel.setBrokerState(brokerId, Broker.State.BAD_DISKS);
         }
       }
+
+      // Populate disk information for the cluster model.
+      if (populateDiskInfo) {
```

Comment:
For readability, can we move this part to a separate function and call that function from here?